src-electron/db/query-endpoint-type.js

```diff
@@ -200,9 +201,11 @@ ON
 WHERE
   ENDPOINT_TYPE_CLUSTER.ENDPOINT_TYPE_REF IN (${endpointTypeIds})
 AND
-  ENDPOINT_TYPE_CLUSTER.SIDE IS NOT "" AND ENDPOINT_TYPE_CLUSTER.ENABLED=1
+  ENDPOINT_TYPE_CLUSTER.SIDE IS NOT "" AND ENDPOINT_TYPE_CLUSTER.ENABLED = 1
 GROUP BY
-  NAME, SIDE`
+  NAME, SIDE
+ORDER BY 
+  CLUSTER.MANUFACTURER_CODE, CLUSTER.CODE, CLUSTER.DEFINE`
```

Comment:
Again, it would be very helpful on the Matter side if this ordering change were a separate changeset from everything else.

Reply:
Ok. I'll split this up. Hold on.